@xen-orchestra/lite/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## **next**
 
+- Update UiCounter component (PR [#7438](https://github.com/vatesfr/xen-orchestra/pull/7438))
```

Comment:
Let's keep the changelog user focused. If there's no visible change from a user point of view, no need to add a changelog entry.